### ISSUE TITLE
Variable Reference don't works

### ISSUE THREAD

**Issue by Lucasb5800** [a85/POSTMan-Chrome-Extension#11643]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

The problem is that I generate a request with a variable inside and then when I change a value of the variable it stops working and the value is not recognized in the request itself.

### Steps To Reproduce

1- Generate a new request with a variable
2- Change the value of variable or move it of the position in the list
3- Send the reques and this will not works

### Screenshots or Videos

![Captura de Error Body](https://user-images.githubusercontent.com/90205338/213443939-95b91924-0950-4837-bdcc-f01e546f5cd1.jpg)
![Captura de Peticion](https://user-images.githubusercontent.com/90205338/213443944-24715eb8-18b4-4428-8431-2c260fc1fead.jpg)
![Captura de Variables](https://user-images.githubusercontent.com/90205338/213443947-222a439f-0226-401f-be02-9f5c91da2adb.jpg)

### Operating System

Windows

### Postman Version

10.8.0

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
@Lucasb5800 after step 2, can you confirm if you saved the environment?
This seems to be a problem with variable resolution itself. 
Can you also over over the variables in the UI and tell me what values you see? 
<img width="522" alt="image" src="https://user-images.githubusercontent.com/1040583/213452551-27dd76eb-41f0-4cde-9a74-54338b1c43df.png">

**Comment by Lucasb5800** [comment]
Yes, after step 2 confirm the changes and then send the request. The variable capture is as follows:
![Captura de Variables En Peticion](https://user-images.githubusercontent.com/90205338/213457246-e3ecee77-d3eb-4012-9b48-ca2528c754c7.jpg)

**Comment by giridharvc7** [comment]
Your current values are empty, go to your environment ensure the current values are appropriately set. If you find all your variables don't have current values,  choose `Reset all` to copy the inital value to the current value
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/1040583/213493932-be447242-8986-401e-9a8c-4a43e67281b9.png">
Let us know if this works
